src/main.rs

```diff
@@ -40,11 +40,15 @@ fn codeowners_enforcer(
         override_builder.add(&format!("!{}", ignore_pattern))?;
     }
 
+    // Always ignore .git directory
+    override_builder.add("!.git/*")?;
```

Comment:
When I first implemented just `.hidden(false)`, I noticed that the `.git` directory was showing up as not covered by the codeowners file. In reading [the documentation](https://docs.rs/ignore/0.1.4/ignore/struct.WalkBuilder.html) for `ignore::WalkBuilder`, I see that they're reading `.gitignore`, but it's non-standard to add `.git` to the `.gitignore` file. Therefore, I explicitly ignore it here (as it's not supported by `CODEOWNERS` anyway).

Reply:
I havent used this tool in a long time so I don't remember. Do you need it to be `!.git/*` or could it just be `!.git/`

Reply:
Yep, `!.git/` works. I made that change.